camel/models/model_factory.py

```diff
@@ -50,8 +55,10 @@ def create(model_type: ModelType,
             model_class = OpenAIModel
         elif model_type == ModelType.STUB:
             model_class = StubModel
+        elif model_type.is_open_source:
+            model_class = OpenSourceModel
```

Comment:
Should we also add `model_type.is_openai` as well?

Reply:
Sure. I have added one in `typing.py`.